STX.REST.RESTFulSense.Clients/Models/Services/HttpExchanges/Headers/RangeConditionHeader.cs

```diff
@@ -8,7 +8,7 @@ namespace STX.REST.RESTFulSense.Clients.Models.Services.HttpExchanges.Headers
 {
     public class RangeConditionHeader
     {
-        public DateTimeOffset Date { get; init; }
+        public DateTimeOffset? Date { get; init; }
```

Comment:
Any reason why was this changed to `public DateTimeOffset? Date { get; init; }`

We have the project setup as `<Nullable>disable</Nullable>`

Reply:
DateTimeOffset is a value type, it´s optional for RangeConditionHeader, so it should allow null.